File: app/src/main/java/ru/dgis/doublegistiles/model/FirmDto.java
package ru.dgis.doublegistiles.model;

import com.google.gson.annotations.SerializedName;

public class FirmDto {

    @SerializedName("id")
    private long id;
    @SerializedName("name")
    private String name;
    @SerializedName("floor")
    private String floor;
    @SerializedName("main_rubric")
    private String mainRubric;

    public FirmDto(long id, String name, String floor, String mainRubric) {
        this.id = id;
        this.name = name;
        this.floor = floor;
        this.mainRubric = mainRubric;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getFloor() {
        return floor;
    }

    public String getMainRubric() {
        return mainRubric;
    }
}

File: app/src/main/java/ru/dgis/doublegistiles/model/FirmDto.kt
package ru.dgis.doublegistiles.model

import com.google.gson.annotations.SerializedName

class FirmDto(@field:SerializedName("id")
              val id: Long, @field:SerializedName("name")
              val name: String, @field:SerializedName("floor")
              val floor: String, @field:SerializedName("main_rubric")
              val mainRubric: String)
